master/internal/experiment_filter.go

```diff
@@ -361,9 +361,16 @@ func (e experimentFilter) toSQL(q *bun.SelectQuery,
 			case doesNotContain:
 				queryString = "? NOT ILIKE ?"
 				queryArgs = append(queryArgs, bun.Safe(col), fmt.Sprintf("%%%s%%", *e.Value))
-			case empty, notEmpty:
-				queryString = "? ?"
+			case empty:
+				queryString = "? ? OR ? ? ? OR ? ? ?"
+				queryArgs = append(queryArgs, bun.Safe(col), bun.Safe(oSQL))
```

Comment:
once you are in this empty case, just writing out the fragment is significantly more readable to me
```suggestion
				queryString = "? IS NULL OR ? = "" OR ? = []"
				queryArgs = append(queryArgs, bun.Safe(col), bun.Safe(col), bun.Safe(col))
```

Reply:
i know that goes against what is happening here already, so i'm not blocking, just giving my 2c.

Reply:
this does look a lot neater ✅